apps/resource_manager/lib/identities/schemas/user.ex

```diff
@@ -15,17 +15,18 @@ defmodule ResourceManager.Identities.Schemas.User do
 
   @typedoc "User schema fields"
   @type t :: %__MODULE__{
-          id: binary(),
+          id: Ecto.UUID.t(),
           username: String.t(),
           status: String.t(),
-          password: Password.t(),
+          password: Password.t() | Ecto.Association.NotLoaded.t(),
```

Comment:
Is this change really needed?

Reply:
Actually this  is just  a more precise way to spec an schema because ecto and repo do not preload the fields by default and the return is exactly this not loaded struct.